decidim-core/app/commands/decidim/update_account.rb

```diff
@@ -48,7 +48,7 @@ def event_arguments
     def update_personal_data
       current_user.locale = @form.locale
       current_user.name = @form.name
-      current_user.nickname = @form.nickname
+      current_user.nickname = @form.nickname&.downcase
```

Comment:
I think this is no longer necessary with the validation
```suggestion
      current_user.nickname = @form.nickname
```